packages/integration/src/gitlab/config.ts

```diff
@@ -68,8 +67,6 @@ export function readGitLabIntegrationConfig(
 
   if (apiBaseUrl) {
     apiBaseUrl = apiBaseUrl.replace(/\/+$/, '');
-  } else if (host === GITLAB_HOST) {
-    apiBaseUrl = GITLAB_API_BASE_URL;
```

Comment:
I think this should not be removed (but you do need to add `https://` to the constant, that is a bug). The point here is that you will be able to add just a token for public gitlab if you wanted to, without having to figure out and fill in their public API base URL.

```
integrations:
  gitlab:
    host: gitlab.com
    token: blah
```

see the documented behavior here https://github.com/backstage/backstage/blob/master/packages/integration/src/gitlab/config.ts#L36

Reply:
It's the case that you can just insert the token. The constant is incorrect.